src/ukify/ukify.py

```diff
@@ -1797,6 +1797,9 @@ def finalize_options(opts):
     if opts.sign_kernel and not opts.sb_key and not opts.sb_cert_name:
         raise ValueError('--sign-kernel requires either --secureboot-private-key= and --secureboot-certificate= (for sbsign) or --secureboot-certificate-name= (for pesign) to be specified')
 
+    if opts.profile and opts.profile.startswith('@'):
+        opts.profile = pathlib.Path(opts.profile[1:])
```

Comment:
this seems to be a pattern by now, maybe add a helper?

`opts.profile = resolve_at_path(opts.profile)`

or so?